lecture/src/test/java/de/tum/in/www1/artemis/lecture/AttachmentUnitIntegrationTest.java

```diff
@@ -82,8 +82,12 @@ public void createAttachmentUnit_asInstructor_shouldCreateAttachmentUnit() throw
         var persistedAttachmentUnit = request.postWithResponseBody("/api/lectures/" + this.lecture1.getId() + "/attachment-units", attachmentUnit, AttachmentUnit.class,
                 HttpStatus.CREATED);
         assertThat(persistedAttachmentUnit.getId()).isNotNull();
+        this.attachment.setAttachmentUnit(persistedAttachmentUnit);
+        var persistedAttachment = attachmentRepository.save(attachment);
```

Comment:
If you don't call the endpoint, you can remove it though. You can see that the assertions are very trivial.
e.g. line 85 and 88 or 84 and 87

Reply:
Yeah, it's not possible to call it. Alright, I will remove it, both are fine for me